doc/cygnus-ngsi/flume_extensions_catalogue/README.md

```diff
@@ -12,6 +12,7 @@
     * [NGSIMongoSink](./ngsi_mongo_sink.md)
     * [NGSIMySQLSink](./ngsi_mysql_sink.md)
     * [NGSIPostgreSQLSink](./ngsi_postgresql_sink.md)
+    * [NGSIPostGISSink](./ngsi_postgis_sink.md)
     * [NGSISTHSink](./ngsi_sth_sink.md)
     * [NGSIOrionSink](./ngsi_orion_sink.md)
     * [NGSITestSink](./ngsi_test_sink.md)
```

Comment:
Apart from including the new sink in this file you should also modify the ones shown in this places:

* https://github.com/telefonicaid/fiware-cygnus/pull/1615/files: README.md, cygnus-ngsi/README.md (using 1.12.0 as version), mkdocs.yml
* https://github.com/telefonicaid/fiware-cygnus/pull/1617/files: cygnus-ngsi/README. md again

Reply:
Boot files are already updated

Reply:
![imagen](https://user-images.githubusercontent.com/1534240/55615315-6d778500-578f-11e9-8005-86419ccafbe2.png)

I see modifications in two README.md file, but I think that cygnus-ngsi/README.md (a third one) is pending.

Reply:
@fgalan  cygnus-ngsi/README.md was also updated.